apps/minifront/src/state/index.ts

```diff
@@ -18,9 +20,11 @@ import { createIbcInSlice, IbcInSlice } from './ibc-in';
 enableMapSet();
 
 export interface AllSlices {
```

Comment:
can we add a comment on what _slices_ are how in the context of the `SliceCreator<SliceInterface>` custom type?